SU2_CFD/src/output/CFlowOutput.cpp

```diff
@@ -774,6 +744,108 @@ void CFlowOutput::SetAnalyzeSurface_SpeciesVariance(const CSolver* const*solver,
   SetHistoryOutputValue("SURFACE_SPECIES_VARIANCE", Tot_Surface_SpeciesVariance);
 }
 
+void CFlowOutput::SetCustomOutputs(const CSolver* const* solver, const CGeometry *geometry, const CConfig *config) {
+
+  const bool axisymmetric = config->GetAxisymmetric();
+  const auto* flowNodes = su2staticcast_p<const CFlowVariable*>(solver[FLOW_SOL]->GetNodes());
+
+  for (auto& output : customOutputs) {
+    if (output.varIndices.empty()) {
+      /*--- Setup indices for the symbols in the expression. ---*/
+
+      if (config->GetNEMOProblem()) {
+        ConvertVariableSymbolsToIndices(
+            CNEMOEulerVariable::template CIndices<unsigned long>(nDim, config->GetnSpecies()), output);
+      } else if (config->GetKind_Regime() == ENUM_REGIME::COMPRESSIBLE) {
+        ConvertVariableSymbolsToIndices(CEulerVariable::template CIndices<unsigned long>(nDim, 0), output);
+      } else {
+        ConvertVariableSymbolsToIndices(CIncEulerVariable::template CIndices<unsigned long>(nDim, 0), output);
+      }
```

Comment:
```suggestion
      if (config->GetNEMOProblem()) {
        ConvertVariableSymbolsToIndices(
            CNEMOEulerVariable::template CIndices<unsigned long>(nDim, config->GetnSpecies()), output);
      } else if (config->GetKind_Regime() == ENUM_REGIME::COMPRESSIBLE) {
        ConvertVariableSymbolsToIndices(CEulerVariable::template CIndices<unsigned long>(nDim, 0), output);
      } else if (config->GetKind_Regime() == ENUM_REGIME::INCOMPRESSIBLE) {
        ConvertVariableSymbolsToIndices(CIncEulerVariable::template CIndices<unsigned long>(nDim, 0), output);
      }
```
For some reasons I am not a fan of these else statements that hold the remaining option (that is also why I like switches with enum classes because it is so nice and explicit)... would it be possible to change it like so?